puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/bastion.yaml

```diff
@@ -39,7 +50,8 @@
           url: https://github.com/katello/bastion
     scm:
       - git:
-          url: https://github.com/katello/bastion
+          url: git://github.com/katello/bastion.git
```

Comment:
Could this re-use the SCM macro defined at the top of the file?

Reply:
it uses `master` branch instead of `${branch}`. so i don't think so?

Reply:
instead i just added a `branch` parameter that defaults to `master` so we can use the same scm.